packages/@aws-cdk/aws-stepfunctions-tasks/lib/eks/call.ts

```diff
@@ -0,0 +1,118 @@
+import * as eks from '@aws-cdk/aws-eks';
+import * as iam from '@aws-cdk/aws-iam';
+import * as sfn from '@aws-cdk/aws-stepfunctions';
+import { Construct } from 'constructs';
+import { integrationResourceArn, validatePatternSupported } from '../private/task-utils';
+
+/**
+ * Properties for calling a EKS endpoint with EksCall
+ * @experimental
+ */
+export interface EksCallProps extends sfn.TaskStateBaseProps {
+
+  /**
+   * The EKS cluster
+   */
+  readonly cluster: eks.ICluster;
+
+  /**
+   * HTTP method ("GET", "POST", "PUT", ...) part of HTTP request
+   */
+  readonly httpMethod: HttpMethods;
+
+  /**
+   * HTTP path of the Kubernetes REST API operation
+   */
+  readonly httpPath: string;
+
+  /**
+   * Query Parameters part of HTTP request
```

Comment:
nit: would be helpful to include doc links and or purpose of query parameters. the current doc strings don't add much information from the parameter names

Reply:
Query parameters would look something like this: https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.19/#-strong-write-operations-cronjob-v1beta1-batch-strong- I'm not sure if it would be helpful to include an example such as this

Reply:
gotcha, I agree. maybe using it in the readme example would be helpful for users though.